packages/siwf/src/config.ts

```diff
@@ -1,4 +1,4 @@
-import { Network, ProxyUrl } from './enums';
+import { Network, ProxyUrl } from './enums.js';
```

Comment:
I believe this is required with the version bump

Reply:
If that's the case, maybe enable an additional eslint rule?
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md

Reply:
I've had lots of trouble getting that one to work. Perhaps for the v2 side I can get it working with less nesting.

Reply:
Also, possibly the extensions are not 100% strictly required when using a bundler, since the output JS is a monolithic file. But probably good practice anyway.